docs/pages/machine-workload-identity/mwi-use-cases/mwi-ci-cd.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: CI/CD with Machine & Workload Identity
+description: Configuring CI/CD with Machine & Workload Identity
+labels:
+ - conceptual
+ - mwi
+---
+
+Teleport Machine & Workload Identity eliminates the need for long-lived static secrets in CI/CD pipelines by issuing short-lived, automatically generated certificates at runtime.
+Teleport natively integrates with many CI/CD providers and deployment targets. Refer to the links at the bottom of this page for instructions on using Machine & Workload Identity with your preferred tools.
```

Comment:
Do we want people to refer to links at the bottom of the page, or do the tiles in the very next section?